src/pybind/mgr/dashboard/ci/cephadm/ceph_cluster.yml

```
@@ -36,7 +36,18 @@ parameters:
  files:
   - bootstrap-cluster.sh
  cmds:
+ # updating the dnf.conf to make the dnf faster
+ - echo -e "[main]\nfastestmirror=true\nmax_parallel_downloads=10\nmetadata_expire=1h\nclean_requirements_on_remove=true\nassumeyes=true\ngpgcheck=1\nkeepcache=0\nplugins=1\ninstallonly_limit=3" | sudo tee /etc/dnf/dnf.conf
```

Let's try to make things more readable when possible (yaml [supports multiple multiline syntaxes](https://yaml-multiline.info/)).
```suggestion
 - |
   echo "\
[main]
fastestmirror=true
max_parallel_downloads=10
metadata_expire=1h
...
```
Or even better, add this as separate file to the `files` section. Embedding different languages (Jinja + YAML + bash + systemd.conf) makes this hard to follow.

---

thanks, i separated them out.